sources/platform/workflow/config.md

```diff
@@ -402,7 +402,7 @@ Jobs are the executable units of your Assembly Line.  A list of all supported jo
         * `replicate` -- an optional setting that allows you to copy the current version of an `IN` resource in the context of the job to the desired `OUT` resource. This is useful if you need some pre-processing, for example, to validate the commit message of a git commit before you execute the actual job. A word of caution: this, if used improperly, can lead to unexpected behavior in your workflow
         * `replicateOnPullRequest` -- an optional setting that can be used with replicate, specify true to update the replicated `OUT` resource on pull requests.
         * `overwrite` -- an optional setting that allows you to completely replace the state of the `OUT` resource. This is useful, for example, if you have multiple jobs that affect the state of the resource and you want the state of your resource to always reflect the most recent update. The default value is `false`, which means the state of your resource is always appended to by the `OUT`, never replaced.
-
+        * `on_failure` -- an optional setting, if set to true, OUT version is processed even on job failure. The default behavior is false in which case OUT version is processed only if job is successful.
```

Comment:
The language should be consistent with the other optional settings on this page. Take a look at `replicate` and `overwrite`, for example.